src/Microsoft.ML.Tokenizers/Model/Tiktoken.cs

```diff
@@ -112,7 +112,7 @@ private Tiktoken(Stream vocabStream, IReadOnlyDictionary<string, int>? specialTo
         /// <param name="cacheSize">The size of the cache to use.</param>
         /// <param name="normalizer">To normalize the text before tokenization</param>
         /// <returns>The tokenizer</returns>
-        public static Tokenizer CreateByModelName(
+        public static Tokenizer CreateTokenizerByModelName(
```

Comment:
Should "By" instead be "For"? And is the "Name" part needed? I'm wondering if it should be "CreateTokenizerForModel" and "CreateTokenizerForModelAsync"

Reply:
CreateTokenizerForModel/Async looks good to me.